Nightly Python runs are broken

@amueller, FYI it seems that #154 may have broken the nightly test runs:

```txt
  docker exec --workdir /src/MLOS/source/Mlos.Python mlos-build-ubuntu-20.04 \
    python3.7 -m pytest -sv mlos/Optimizers/long_haul_test/*
  shell: /bin/bash -e {0}
============================= test session starts ==============================
platform linux -- Python 3.7.9, pytest-6.1.2, py-1.9.0, pluggy-0.13.1 -- /usr/bin/python3.7
cachedir: .pytest_cache
rootdir: /src/MLOS, configfile: pytest.ini
ERROR: file or directory not found: mlos/Optimizers/long_haul_test/*
collecting ... collected 0 items


============================ no tests ran in 0.00s =============================
Error: Process completed with exit code 4.
```
https://github.com/microsoft/MLOS/runs/1428237005?check_suite_focus=true

No one noticed this because those tests were failing due to timeouts nightly anyways.

@byte-sculptor what do you want to do with nightly?  With it failing every night due to timeouts and no one having time to look at it, it doesn't seem to be super useful right now.

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -119,7 +119,7 @@ jobs:
       timeout-minutes: 700
       run: |
         docker exec --workdir /src/MLOS/source/Mlos.Python mlos-build-ubuntu-${{ matrix.UbuntuVersion }} \
-          python3.7 -m pytest -sv mlos/Optimizers/long_haul_test/*
+          python3.7 -m pytest -sv mlos/Optimizers/long_haul_tests/*.py
     - name: Cleanup docker instance for Ubuntu ${{ matrix.UbuntuVersion }}
       shell: bash
       run: |

